tests/test_compile_error.rs

```diff
@@ -32,7 +32,11 @@ fn test_compile_errors() {
     t.compile_fail("tests/ui/invalid_pyfunctions.rs");
     t.compile_fail("tests/ui/invalid_pymethods.rs");
     // output changes with async feature
-    #[cfg(all(Py_LIMITED_API, feature = "experimental-async"))]
+    #[cfg(all(
+        Py_LIMITED_API,
+        feature = "experimental-async",
+        not(diagnostic_namespace)
```

Comment:
Do we need the feature gate here? I think at this point we've decided that the trybuild tests only work on stable.

Reply:
Right, so I think we don't need the `not(diagnostic_namespace)` gate?

Reply:
Oh, I hadn't realized we don't run these on msrv anymore.